### ISSUE TITLE
repo-review

### ISSUE THREAD

**Issue by knaaptime** [pysal/submodule_template#43]
another tool from our friends at scientific-python to keep an eye on:

https://github.com/scientific-python/repo-review

if i understand, this should check to see if the submodules are keeping in-sync with the submodule template. Could be super useful

**Comment by martinfleis** [comment]
Yes, I was talking about this with folks at the Scientific Python summit. It would be super cool to ensure our stack is in sync.
